dasharo-compatibility/dcu.robot

```diff
@@ -158,5 +156,46 @@ Prepare DCU Test Environment
     ...    6e5a6722955e4f78d947654630f27ff833703fbc04776ffed963c96617f6bb2a
 
     Run    cp ${FW_FILE} dcu/coreboot.rom
+    Run    chmod -R a+rw dcu
     ${local_path}=    Join Path    ${DL_CACHE_DIR}    logo.bmp
     Run    cp ${local_path} dcu/logo.bmp
+
+Write SMMSTORE State 1
+    Dcu.Set UEFI Option    NetworkBoot    Disabled
+
+Check SMMSTORE State 1 (FW)
+    @{option_path}=    Option Name To UEFI Path    NetworkBoot
+    Power On
+    ${menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${path_len}=    Get Length    ${option_path}
+    FOR    ${i}    IN RANGE    ${path_len} - 1
+        ${menu}=    Enter Submenu From Snapshot And Return Construction
+        ...    ${menu}
+        ...    ${option_path[${i}]}
+    END
+    ${state}=    Get Option State    ${menu}    ${option_path[${path_len}-1]}
+    Should Be Equal    ${state}    ${False}
+
+Check SMMSTORE State 1 (DCU)
+    ${result}=    Dcu.Get UEFI Option    NetworkBoot
+    Should Be Equal    ${result}    Disabled
+
+Write SMMSTORE State 2
+    Dcu.Set UEFI Option    NetworkBoot    Enabled
```

Comment:
I like the idea that you try to avoid duplication from the dcu lib.
I do not like the idea that this is 4th tests in this suite, where 3 first tests use one way of using dcu, and you propose another way in 4th case, without changing the prior ones. We need to maintain consistency in implementation across test cases.

Reply:
I am also not a fan of this solution which relies on including DCU options lib here, even if the platform may include another options lib already.

Reply:
The tests are now implemented using the new `lib/dcu.robot`. The `lib/options/dcu.robot` also uses the new lib.